quadriga/src/main/java/edu/asu/diging/quadriga/core/model/Collection.java

```diff
@@ -35,5 +38,13 @@ public String getDescription() {
     public void setDescription(String description) {
         this.description = description;
     }
+    
+    public List<String> getApps() {
+        return apps;
+    }
+
+    public void setApps(List<String> apps) {
+        this.apps = apps;
+    }
```

Comment:
the apps property needs some javadoc. is this the id of the app in citesphere? its name? something else?